tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -218,5 +221,38 @@ internal static string GetLogDirectory()
 
             return logDirectory;
         }
+
+        internal static void CleanLogFiles(string logsDirectory)
+        {
+            var logDaysLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogFileRetentionDays);
```

Comment:
This will make it so this setting can only be set via environment variables. Don't we want to support other configuration options like `web.config` or `datadog.json`?

Reply:
Maybe this is a restriction of our logging system, and that's fine. But then you don't need the `TracerSettings.LogFileRetentionDays` property.

Reply:
Yes, I think right now it requires env vars, because it's catch 22 with the settings that are loaded from elsewhere (they need to instantiate a logger). Also agree with `TracerSettings.LogFileRetentionDays` in that case 👍